src/runtime/thread_pool.cc

```diff
@@ -69,14 +69,14 @@ class ParallelLauncher {
       tvm::runtime::threading::Yield();
     }
     if (!has_error_.load()) return 0;
-    std::string err("");
+    std::stringstream err;
```

Comment:
this is intended to use string, due to security issues raised in SGX backend, please revert this change and add a comment about it

Reply:
std::stringstream was banned by sgx backend and we need it here

Reply:
I reverted this change and add a comment. Then I changed APP_STL to avoid the building error. Please review again.